src/test/java/com/auth0/net/BaseRequestTest.java

```diff
@@ -0,0 +1,122 @@
+package com.auth0.net;
+
+import com.auth0.exception.APIException;
+import com.auth0.exception.Auth0Exception;
+import com.auth0.exception.RateLimitException;
+import okhttp3.Call;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.util.HashMap;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+public class BaseRequestTest {
+
+    private Response response;
+    private Call call;
+    private OkHttpClient client;
+
+    @Before
+    public void setUp() throws Exception {
+        response = Mockito.mock(Response.class);
+
+        call = Mockito.mock(Call.class);
+        Mockito.when(call.execute()).thenReturn(response);
```

Comment:
try statically importing this Mockito methods so you don't repeat `Mockito.` every time

Reply:
Bad habit of mine, will do.